core/src/main/scala/filodb.core/downsample/OffHeapMemory.scala

```diff
@@ -14,7 +15,7 @@ class OffHeapMemory(schemas: Seq[Schema],
   extends StrictLogging {
 
   private val blockMemSize = storeConfig.shardMemSize
-  private val nativeMemSize = storeConfig.ingestionBufferMemSize
+  private val nativeMemSize = systemConfig.getMemorySize("filodb.memstore.ingestion-buffer-mem-size").toBytes
```

Comment:
Decide whether you want to use same config. We could also use a different config for downsampler. I seem to prefer latter. If we go this route, previous comment is canceled.

Reply:
@vishramachandran as discussed this config is not applicable for downsample cluster